packages/pds-ember/public/metadata.json

```diff
@@ -0,0 +1,5 @@
+{
+  "versions": {
+    "main": "https://structure.hashicorp.vercel.app"
+  }
+}
```

Comment:
This is necessary for the version picker - later we can look at creating branches with our release tags so we could like to versioned storybook.

The format is here:
https://github.com/storybookjs/storybook/issues/12202#issuecomment-678864851